a3/skeleton/phonebook_skeleton.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+import sys
+import os
+
+PHONEBOOK_ENTRIES = ".python_phonebook_entries"
+
+
+def main():
+    if len(sys.argv) < 2:
+        exit(1)
+
+    elif sys.argv[1] == "new":
+        # YOUR CODE HERE #
+
+    elif sys.argv[1] == "list":
+        if not os.path.isfile(PHONEBOOK_ENTRIES) or os.path.getsize(
+                PHONEBOOK_ENTRIES) == 0:
+            print("phonebook is empty")
+        else:
+            # YOUR CODE HERE #
+
+    elif sys.argv[1] == "remove":
+        name = " ".join(sys.argv[2:4])
```

Comment:
this will break if someone tries to use anything besides a 2-unit name, probably better to do something like sys.argv[2:-1]